pkg/fleet/bootstraper/bootstraper.go

```diff
@@ -101,9 +105,20 @@ func Bootstrap(ctx context.Context, opts ...Option) error {
 	if downloadedPackage.Name != installerPackage {
 		return fmt.Errorf("unexpected package name: %s, expected %s", downloadedPackage.Name, installerPackage)
 	}
+	hash, err := downloadedPackage.Image.Digest()
+	if err != nil {
+		return fmt.Errorf("failed to get image digest: %w", err)
+	}
+	existingHash, err := os.ReadFile(hashFilePath)
+	if err != nil && !os.IsNotExist(err) {
+		return fmt.Errorf("failed to read hash file: %w", err)
+	}
+	if string(existingHash) == hash.String() {
+		return nil
+	}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ Question:** ‏Do we want a `force` option?